src/Compilers/CSharp/Test/Semantic/Semantics/PropertyFieldKeywordTests.cs

```diff
@@ -1088,6 +1205,312 @@ .maxstack 8
+  IL_000d:  ret
+}
+";
+            }
+            var accessorBindingData = new SourcePropertySymbolBase.AccessorBindingData();
+            comp.TestOnlyCompilationData = accessorBindingData;
+            CompileAndVerify(comp).VerifyDiagnostics().VerifyIL("C.P.set", @"
+{
+  // Code size       10 (0xa)
+  .maxstack  3
+  IL_0000:  ldarg.0
+  IL_0001:  ldarg.1
+  IL_0002:  ldc.i4.2
+  IL_0003:  mul
+  IL_0004:  stfld      ""int C.<P>k__BackingField""
+  IL_0009:  ret
+}
+").VerifyIL("C..ctor", ctorExpectedIL);
+            Assert.Equal(0, accessorBindingData.NumberOfPerformedAccessorBinding);
+        }
+
+        [Theory]
+        //[InlineData("get; set => field = value * 2;")] PROTOTYPE(semi-auto-props): Not yet supported.
+        [InlineData("get => field; set => field = value * 2;")]
+        public void TestAssigningFromConstructorThroughSetterWithFieldKeyword_HasGetter(string accessors)
+        {
+            var comp = CreateCompilation($@"
+public class C
+{{
+    public C()
+    {{
+        P = 5;
+    }}
+
+    public int P {{ {accessors} }}
+
+    public static void Main()
+    {{
+        System.Console.WriteLine(new C().P);
+    }}
+}}
+", options: TestOptions.DebugExe);
+            var accessorBindingData = new SourcePropertySymbolBase.AccessorBindingData();
+            comp.TestOnlyCompilationData = accessorBindingData;
+            CompileAndVerify(comp, expectedOutput: "10").VerifyDiagnostics().VerifyIL("C.P.get", @"
+{
+  // Code size        7 (0x7)
+  .maxstack  1
+  IL_0000:  ldarg.0
+  IL_0001:  ldfld      ""int C.<P>k__BackingField""
+  IL_0006:  ret
+}
+").VerifyIL("C..ctor", @"
+{
+  // Code size       17 (0x11)
+  .maxstack  2
+  IL_0000:  ldarg.0
+  IL_0001:  call       ""object..ctor()""
+  IL_0006:  nop
+  IL_0007:  nop
+  IL_0008:  ldarg.0
+  IL_0009:  ldc.i4.5
+  IL_000a:  call       ""void C.P.set""
+  IL_000f:  nop
+  IL_0010:  ret
+}
+");
+            Assert.Equal(0, accessorBindingData.NumberOfPerformedAccessorBinding);
+        }
+
+        [Theory(Skip = "PROTOTYPE(semi-auto-props): Not supported yet.")]
+        [InlineData("get; set => _ = value;")]
+        [InlineData("set => _ = value;")]
+        [InlineData("get => field; set => _ = value;")]
+        public void TestAssigningFromConstructorThroughSetter_RegularSetter(string accessors)
+        {
+            var comp = CreateCompilation($@"
+public class C
+{{
+    public C()
+    {{
+        P = 5;
+    }}
+
+    public int P {{ {accessors} }}
+
+    public static void Main()
+    {{
+        System.Console.WriteLine(new C().P);
+    }}
+}}
+", options: TestOptions.DebugExe);
+            var accessorBindingData = new SourcePropertySymbolBase.AccessorBindingData();
+            comp.TestOnlyCompilationData = accessorBindingData;
+            CompileAndVerify(comp, expectedOutput: "0").VerifyDiagnostics().VerifyIL("C.P.get", @"
+").VerifyIL("C..ctor", @"
+");
+            Assert.Equal(1, accessorBindingData.NumberOfPerformedAccessorBinding);
```

Comment:
@AlekseyTs This test is skipped for now with a PROTOTYPE. It's likely this will be zero when it's unskipped. Otherwise, I'll investigate why it's not zero.

Reply:
> This test is skipped for now with a PROTOTYPE.

Consider changing the expectation to 0 then.